src/Utils.vala

```diff
@@ -62,6 +62,7 @@ public class Power.Utils {
             total_x_delta = 0.0;
             Power.Services.DeviceManager.get_default ()
                 .change_brightness ((int) (Math.round (dir) * BRIGHTNESS_STEP));
+                warning (dir.to_string ());
```

Comment:
I think this should be a "debug" message as it is not indicative of any potential problem.

Reply:
I think I accidently left this after testing how smooth scrolling works